TIG/TIG/Extension/Int+Ext.swift

```diff
@@ -29,6 +29,25 @@ extension Int {
     }
     return result
   }
+    
+    // 0시간 0분도 표시
+      func formattedFullDuration() -> String {
+          let totalMinutes = self * 30
+          let hours = totalMinutes / 60
+          let minutes = totalMinutes % 60
+          
+          var result = ""
+          if hours >= 0 {
+            result += "\(hours)시간"
+          }
+          if minutes >= 0 {
+            if !result.isEmpty {
+              result += " "
+            }
+            result += "\(minutes)분"
+          }
+          return result
```

Comment:
```suggestion
         return "\(hour)시간 \(minute)분"
```
바로 리터럴 값으로 리턴해줘도 되지 않을까요?
보면 hour, minute 모두 0이상일 때의 조건인데, 각 값이 -인 경우가 없어보여서 조건문을 넣을 필요가 없어보입니다

Reply:
수정하였습니다!